### PR TITLE
Add the IBM Power HMC provider repo

### PR DESCRIPTION
**Pull request by agrare**
https://github.com/ManageIQ/manageiq-providers-ibm_power_hmc

### CODE REVIEW

File: config/labels.yml [hunk 1]
```diff
@@ -254,6 +254,10 @@ repos:
     <<: *repo_introduced_in_kasparov
     <<: *provider_plugin
     <<: *release
+  ManageIQ/manageiq-providers-ibm_power_hmc:
+    <<: *common
```
Comment (Fryguy) [inline config/labels.yml:258]:
```suggestion
    <<: *common
    # <<: *repo_introduced_in_najdorf
```

Comment (Fryguy) [reply config/labels.yml:258]: This helps me later when we cut najdorf

**Comment by Fryguy** [comment]
hehe, I had this locally, but you beat me to it

File: config/labels.yml (hunk 1 above)
Comment (agrare) [reply config/labels.yml:258]: :+1:

**Pull request merged**